app/models/review.rb

```diff
@@ -0,0 +1,42 @@
+class Review < ActiveRecord::Base
+  belongs_to :appointment # связь с должностями
+  belongs_to :university, foreign_key: :university_id
+  belongs_to :university_author, class_name: University, foreign_key: :university_auth_id
+
+  #validates :contract_date, presence: true
+  #validates :sheet_number, presence: true
+
+  enum status: {    # статус рецензии
+      обработка: 0,
+      завершен: 1,
+      истек: 2
+  }
+  enum ordt: {    # тип заказа
+      физ: 0,
+      юр: 1
+  }
+  enum evaluation: {    # тип отзыва на рецензию
+      '+' => 0,
+      '-' => 1,
+      неопр: 2
+  }
+  enum paid: {    # оплата
+      да: 0,
+      нет: 1
+  }
+  scope :keyword_search, -> (params) { where("title LIKE ? OR author LIKE ? OR number_review LIKE ? OR contract_number LIKE ? OR total_cost LIKE ? OR cost LIKE ?", "%#{params[:search_keywords]}%", "%#{params[:search_keywords]}%", "%#{params[:search_keywords]}%", "%#{params[:search_keywords]}%", "#{params[:search_keywords]}%", "#{params[:search_keywords]}%") }
+  scope :date_search, -> (params) { where(contract_date: Date.parse(params[:start_date])..Date.parse(params[:end_date])) }
+  scope :university_search, ->(params) {
+    joins('LEFT JOIN universities AS u ON u.id = university_id')
+    .joins('LEFT JOIN universities AS u_a ON u_a.id = university_auth_id')
+    .where('u.name LIKE ? OR u_a.name LIKE ?', "%#{params[:search_keywords]}%", "%#{params[:search_keywords]}%") }
```

Comment:
Line is too long. [114/80]<br>Expression at 32, 114 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.